target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-fritzbox-7530.dts

```diff
@@ -17,7 +17,7 @@
 		led-boot = &power_green;
 		led-failsafe = &info_red;
 		led-running = &power_green;
-		led-upgrade = &info_green;
+		led-upgrade = &info_red;
```

Comment:
This change looks unrelated to adding support for VRX518

Reply:
It clears the green LED for DSL status indicator, which is added in the same patch. @janh did this, I assume to match the behavior of the vendor fw

Reply:
Actually, mostly because the LEDs are labeled on the case, which I removed :)

Reply:
The vendor firmware uses the power LED (labeled "Power/DSL") for DSL status, and according to the manual the green info LED is used as upgrade indicator.

But that setup isn't ideal, as it makes the power LED ambiguous. And other Fritzboxes already use the green info LED for DSL status in OpenWrt, and (except for 7430) one of the red LEDs as upgrade indicator, so it seems like a better idea to match that.